app/main.py

```diff
@@ -2,0 +1,11 @@
+def copy_file(command: str) -> None:
+    data = command.split()
+    file_to_copy = data[1]
+    copy = data[2]
+
+    if len(data) != 3 or data[0] != "cp" or file_to_copy == copy:
+        return
+
+    with open(file_to_copy, "r") as file_in, open(copy, "a") as file_out:
```

Comment:
There is no error handling for file operations. Consider using a try-except block to handle potential IO errors, such as the file not existing or permission issues.